link.x

```diff
@@ -17,6 +17,14 @@ SECTIONS
     _einterrupts = .;
   } > FLASH
 
+  PROVIDE(_text_start = _einterrupts);
```

Comment:
nit: could you rename _text_start to _start? That's the convention we have for symbols that point to section boundaries.